pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_inference.cc

```diff
@@ -46,6 +46,71 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleLoad(
   API_END_RETURN()
 }
 
+JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleLoad__Ljava_io_InputStream_2_3I_3B(
+   JNIEnv* env, jobject jthis, jobject jis, jintArray jarray, jbyteArray arr) {
+    API_BEGIN()
+    jclass is_class = env->GetObjectClass(jis);
+    if (is_class == nullptr) {
+        env->ThrowNew(NULL_PTR_EXCEPTION_CLASS, "Java inputStream class is not found");
+        return -1;
+    }
+    jmethodID method_id = env->GetMethodID(is_class, "read", "([BII)I");
+    if (method_id == nullptr) {
+        env->ThrowNew(ENGINE_EXCEPTION_CLASS, "The read method in InputStream is not found");
+        return -1;
+    }
+    std::ostringstream os;
+    int len = env->GetArrayLength(arr);
+    char* temp = new char[len];
```

Comment:
If it is temporary memory, just put it in stack 
```suggestion
    char temp[len];
```